app/decorators/mod_action_decorator.rb

```diff
@@ -295,7 +295,11 @@ def format_description
     when "wiki_page_unlock"
       "Unlocked wiki page [[#{vals['wiki_page']}]]"
     when "wiki_page_rename"
-      "Renamed wiki page ([[#{vals['old_title']}]] → [[#{vals['new_title']}]])"
+      if vals['old_title'].nil?
```

Comment:
Wiki page creation logging appears to be a bug instead of a feature. Wikis have a history so their creator is known, so it isn't as important to log them to the action log. As only moderators, and in the case of artists, janitors, can destroy these entities, there is sufficient logging to investigate the source of them. Updating the logging action to fire only on update instead of on creation would fix this and remove the need for the updated decorators.

You can use `new_record?` to test if a record has been saved or not in AR callbacks.

Reply:
Makes sense, the action is named rename after all. I removed the decorator code with special handling when it was created, the old logactions will still say created but I think thats ok